docs/resources/apig_channel.md

```diff
@@ -314,7 +314,13 @@ The `cce_config` block supports:
   + **statefulset**: Stateful load.
   + **daemonset**: Daemons set.
 
-* `workload_name` - (Required, String) Specifies the workload name.
+* `workload_name` - (Optional, String) Specifies the workload name.
+
+* `label_key` - (Optional, String) Specifies the service label key.
+  One of this parameter and `workload_name` must be set.
```

Comment:
```suggestion
  One of this parameter or `workload_name` must be set.
```

Reply:
the parameter `workload_name` has been removed from the document.